### ISSUE TITLE
Suggestions on layout

### ISSUE THREAD

**Issue by CarolinaFernandez** [torsec/shield-h2020#1]
It would be nice to include some extra media resources on the site:

* Some background in the "SHIELD" square in the [main page](https://torsec.github.io/shield-h2020)
* Consider some light colour scheme (fitting with the logo) at the other squares in the main page
* Partners logo thumbnail also at the [consortium page](https://torsec.github.io/shield-h2020/about/consortium.html)

**Comment by psmiraglia** [comment]
Thanks for the suggestions. We'll do our best...

**Comment by mrcdb** [comment]
As an update, we implemented partners logo thumbnail at the consortium page.
Thanks for your suggestions.

**Comment by mrcdb** [comment]
Hi, all the suggested features were included with commit add31f617d43099351069b4e3ec37e6a822e99b0.

**Comment by CarolinaFernandez** [comment]
Checked, looks nicer. Thanks.

Regarding the colour scheme, the following could be used -- after being properly adapted to sass:

```
h1, h2, h3 {
    color: #3b63a8;
}

.row h3::first-letter {
    font-size: 1.3em;
}
.row h3 {
    text-transform: uppercase;
    font-size: 1.25em;
}

.navbar-default .navbar-nav>.active>a, .navbar-default .navbar-nav>.active>a:hover, .navbar-default .navbar-nav>.active>a:focus {
    background-color: #c9dde7;
}

.navbar-default .navbar-nav>.open>a, .navbar-default .navbar-nav>.open>a:hover, .navbar-default .navbar-nav>.open>a:focus {
    background-color: #c9dde7;
}

.navbar-default {
    background-color: #e1f1f9;
    border-color: #b9cdd7;
}

.dropdown-menu>li>a:hover {
    background-color: #d9edf7;
}

.navbar-default .navbar-nav>.active>a, .navbar-default .navbar-nav>.active>a:hover, .navbar-default .navbar-nav>.active>a:focus {
    color: #3b63a8;
}

.navbar-default .navbar-nav>li>a {
    color: #3f90c2;
}

.navbar-default .navbar-nav>li>a:hover {
    color: #3b63a8;
}

div.col-md-4>h3 {
    font-size: 24px;
    border-bottom: 3px solid #b9cdd7;
}
```
